### DISCUSSION TITLE
Final Presentation Amber & Wyatt

### DISCUSSION THREAD

**Discussion by AmberMcClurg** [birkenkrahe/ai482#16] [General]
https://youtu.be/-OsKFvW14V4

**Comment by birkenkrahe** [comment]
A lot of information in a small space! Enjoyed this but was a
   little overwhelmed in the end. I felt there would have been benefit
   in pulling it together, and perhaps also in presenting a personal
   perspective, including a reflection on the project (how you split
   up the work, if you were still happy with your topic at the end,
   what you took away from it, etc.). Thank you!
   
   * Good overview and interesting sources - could have been more
     focused on AR/VR. Felt a little rushed.
   * Case study focus - need to pull together for a story
   * Good sources
